subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -12,6 +12,15 @@ menuconfig BT_FAST_PAIR
 
 if BT_FAST_PAIR
 
+config BT_FAST_PAIR_SUBSEQUENT_PAIRING
+	bool "Fast Pair Subsequent Pairing"
+	default y if !BT_FAST_PAIR_FMDN
+	select EXPERIMENTAL
+	help
+	  Enable support for the Subsequent Pairing feature. Subsequent pairing
+	  is the later pairing between the previously initial-paired device and
```

Comment:
`Subsequent pairing is the later pairing between a Fast Pair Provider that was initial-paired to your Google account and your another Fast Pair Seeker where the same Google Account is logged-in.`